run_unittests.py

```diff
@@ -3687,6 +3687,16 @@ def test_clang_tidy(self):
         out = self.run_target('clang-tidy')
         self.assertIn('cttest.cpp:4:20', out)
 
+    def test_identity_cross(self):
+        testdir = os.path.join(self.unit_test_dir, '71 cross')
+        # Do a build to generate a cross file where the host is this target
+        self.init(testdir, extra_args=['-Dgenerate=true'])
+        self.meson_cross_file = os.path.join(self.builddir, "crossfile")
+        self.assertTrue(os.path.exists(self.meson_cross_file))
```

Comment:
There should be a `finally` block that deletes the generated crossfile.

Reply:
It goes into the build directory, doesn't that get cleaned up already?

Reply:
D'oh! Of course it does. 🤦‍♂